src/main/web_src/src/service/services/tpsf/TpsfService.js

```diff
@@ -59,6 +59,6 @@ export default {
 	getExcelForIdenter(userArray) {
 		if (!userArray) return
 		const endpoint = `${getTpsfUrl()}/dolly/testdata/excel`
-		return Request.post(endpoint, userArray)
+		return Request.post(endpoint, userArray, ResponseType.TEXT)
```

Comment:
Altså eneste grunnen for at vi har noe som heter fetchJson er at vi gjør det mye.

Må du gå igjennom Request.post ? 
Jeg ville skrvet dennne som bare: 

`api.fetch(url, { method: 'POST', headers: {"Content-Type": "application/csv" } }, data).then(response => ({ data: response.text() }))`

Reply:
Kan bli litt mange consepter